scripts/react-app/opener.js

```diff
@@ -0,0 +1,58 @@
+"use strict";
+let childProcess = require("child_process");
+let os = require("os");
+
+module.exports = function opener(args, options, callback) {
```

Comment:
better to change this to just be a function and export all the functions from this file (see /scripts/common/packageUtil.js). Also I think we should rename this file to `platformUtil.js` and move it into the common folder so it can be reused